do.py

```diff
@@ -150,22 +153,20 @@ def test():
     go_coverage_threshold = 35
     # TODO: not able to run the test from main directory
     os.chdir("pkg")
-    try:
-        run(["go test ./... -v -coverprofile coverage.txt | tee coverage.out"])
-    finally:
-        os.chdir("..")
-
-    with open("pkg/coverage.out") as fp:
-        out = fp.read()
-        result = re.findall(r"coverage:.*\s(\d+)", out)[0]
-        if int(result) < go_coverage_threshold:
-            raise Exception(
-                "Go tests achieved {1}% which is less than Coverage thresold {0}%,".format(
-                    go_coverage_threshold, result))
-        else:
-            print(
-                "Go tests achieved {1}% ,Coverage thresold {0}%".format(
-                    go_coverage_threshold, result))
+    ret = subprocess.run("go test ./... -v -coverprofile coverage.txt".split(), capture_output=True)
```

Comment:
this call also needs to check for exit code (i.e. check=True). moreover, please ensure this works with python2

Reply:
openapiart has python3 only that's when I had added subprocess.run, I have noticed "run" method isn't supported in python2

Reply:
we need to reuse this in snappi repo as well (which needs to support python2)